tsconfig.json

```diff
@@ -4,6 +4,13 @@
 		"declaration": false,
 		"emitDecoratorMetadata": true,
 		"experimentalDecorators": true,
+		"lib": [
+			"dom",
+			"es5",
+			"es2015.iterable",
+			"es2015.symbol",
+			"es2015.symbol.wellknown"
+		],
```

Comment:
Once we land https://github.com/dojo/grunt-dojo2/pull/99 and release we can actually remove the typings file completely (at which point we do need the `"types": [ "intern" ]` - perhaps we should include that in these changes.